lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -107,10 +109,14 @@ def self.run_tests(accumulator)
 
         private
 
+        def self.adapter_class
+          KnapsackPro::Adapters::RSpecAdapter
+        end
+
         def self.log_rspec_command(cli_args, test_file_paths, type)
           case type
           when :subset_queue
-            KnapsackPro.logger.info("To retry the last batch of tests fetched from the API Queue, please run the following command on your machine. (If you use the `-- order random` option, remember to add correct `--seed 123` that you can find at the end of the RSpec output.)")
+            KnapsackPro.logger.info("To retry the last batch of tests fetched from the API Queue, please run the following command on your machine:")
           when :end_of_queue
             KnapsackPro.logger.info("To retry all the tests assigned to this CI node, please run the following command on your machine:")
           end
```

Comment:
L125: I noticed  my mistake from the past. The line slices not enough text.

The logger output is (there is a double space in the `progress  --default-path` string):

```
 [knapsack_pro] bundle exec rspec --order defined --format progress  --default-path spec "spec/controllers/pending_controller_spec.rb"
```

We could fix that.

```ruby
          stringify_cli_args = cli_args.join(' ')
          # there is a space at the end of the string
          stringify_cli_args.slice!("--format #{KnapsackPro::Formatters::RSpecQueueSummaryFormatter} ")
```

so the output would look like this:

```
 INFO -- : [knapsack_pro] bundle exec rspec --order defined --format progress --default-path spec "spec/bar_spec.rb"
```